home_assistant_streamdeck_yaml.py

```diff
@@ -1529,16 +1547,7 @@ async def _handle_key_press(
 ) -> None:
     if not config._is_on:
         turn_on(config, deck, complete_state)
-        if (config.state_entity_id is not None) and (
-            config.state_entity_id.split(".")[0] == "input_boolean"
-        ):
-            await call_service(
-                websocket,
-                "input_boolean.turn_on",
-                {},
-                {"entity_id": config.state_entity_id},
-            )
-        return
```

Comment:
This introduced https://github.com/basnijholt/home-assistant-streamdeck-yaml/issues/186. Fixed in #190